docs/script.rst

```diff
@@ -397,13 +393,12 @@ replaced with an empty String. For example::
     ('Hi Grails users' - wordStartsWithGr) == 'Hi users'
 
 
-
-Remove first match of a word with 5 characters::
+Remove the first 5-character word from a string::
 
     assert ('Remove first match of 5 letter word' - ~/\b\w{5}\b/) == 'Remove  match of 5 letter word'
 
 
-Remove first found numbers followed by a whitespace character::
+Remove the first number with its trailing whitespace from a string::
```

Comment:
It's _all_ whitespace that'll be removed, not just "a whitespace character", right, due to `\s+`?

Reply:
👍